### PR TITLE
[ui] New, bootstrap datetimepicker bump to 4.17.47 

### PR DESCRIPTION
**Pull request by astahlman**
Bump the version of bootstrap datetimepicker to enable support for time-zones. This addresses https://github.com/dpgaspar/Flask-AppBuilder/issues/920.

The quickhowto2 example now includes a `created_at` field on the Contacts model. If you add a filter on this field in the List Contacts view, you can use the dropdown near the top of the page to toggle the time-zone between the browser-local timezone and UTC. The timestamp displayed in the datetime picker widget should reflect the current timezone selection.

### CODE REVIEW

**Comment by astahlman** [comment]
# Local
Screen Shot 2019-03-23 at 11 30 47 AM

# UTC
Screen Shot 2019-03-23 at 11 30 57 AM

**Comment by dpgaspar** [comment]
Hi @astahlman ,

Can you please check why the datetimepicker get's cropped:

image

This is for fieldsets on quickhowto

File: flask_appbuilder/templates/appbuilder/init.html [hunk 1]
```diff
@@ -48,8 +48,10 @@
 
 
 {% block tail_js %}
+<script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.12.0/moment.js"></script>
```
[the remark this answers is not in the document]
Comment (mistercrunch) [reply flask_appbuilder/templates/appbuilder/init.html]: I'd vote for [bad] consistency here and bring in the file to the repo. Even if we have clear/reliable guarantees from `cdnjs.cloudflare.com` (which I doubt we do).

Eventually this should all be `npm` + `webpack`, but in the meantime let's be consistent.

**Comment by dpgaspar** [comment]
Hi @astahlman,

Any news on this?

**Comment by astahlman** [comment]
Hi @dpgaspar, sorry I haven't had time to look into this lately. I'll try and take a look this weekend, but if anyone else wants to take it up before then, feel free.

**Pull request closed without merging**